lib/rspecq/queue.rb

```diff
@@ -241,6 +242,22 @@ def flaky_jobs
       requeued - @redis.hkeys(key_failures)
     end
 
+    def fail_fast_threshold
```

Comment:
This is not used in this commit. Maybe it should be defined in the same commit that uses it.

Reply:
I do not have a strong opinion about the commit/PR distinction. I personally, review the pull request as a whole unit, rather than a per-commit basis. I guess, I can change it

Reply:
3b47fe8